src/functionalTest/resources/features/F-002 - Get Assignments in My Organisation/S-010.td.json

```diff
@@ -0,0 +1,36 @@
+{
+  "_guid_": "S-010",
+
+  "title": "Must return an error response for a missing case ID",
+
+  "specs": [
+    "Richard – with a solicitor role with access to invoke the ‘Get Assignments in My Organisation’ operation of Case Assignment Microservice",
+    "is made by Richard",
+    "does not contain any case ID"
+  ],
+
+  "productName": "Manage Case Assignment Microservice",
+  "operationName": "Get Assignments in My Organisation",
+
+  "method": "GET",
+  "uri": "/case-assignments",
+
+  "s2sClientId": "xui_webapp",
+  "userTokenClientId": "xuiwebapp",
+
+  "users": {
+    "invokingUser": {
+      "username": "befta.master.solicitor.becky@gmail.com",
+      "password": "[[$CCD_BEFTA_MASTER_SOLICITOR_1_PWD]]"
+    }
+  },
```

Comment:
if invoking user is Becky, even this piece of data can be reused from a common object.